server/fleet/app.go

```diff
@@ -1115,6 +1115,9 @@ type ListQueryOptions struct {
 	// MergeInherited merges inherited global queries into the team list.  Is only valid when TeamID
 	// is set.
 	MergeInherited bool
+	// Return queries that are compatible (not a strict check) with this platform. One of "macos",
```

Comment:
@lucasmrod same on the frontend currently, as well as in that `osquery_fleet_schema.json` that defines the compatible platforms per table. I had the URL query param option be `macos` since @rachaelshaw says we want to be moving towards the more user-friendly "macos" for anything user-facing.

When building that compatibility checking logic and adding the db column per our discussion earlier, I can use `darwin` internally but since the URL query param itself will be `"macos"`, I think it makes sense to keep this comment as-is. Wdyt?

Reply:
Sounds good 👍